GiniVision/Assets/de.lproj/Localizable.strings

```diff
@@ -120,3 +120,49 @@
 "ginivision.images.openWithTutorialStep1" = "openWithTutorialStep1";
 "ginivision.images.openWithTutorialStep2" = "openWithTutorialStep2";
 "ginivision.images.openWithTutorialStep3" = "openWithTutorialStep3";
+
+"ginivision.digitalinvoice.screentitle" = "Digital Invoice";
+"ginivision.digitalinvoice.paybuttontitle.noinvoice" = "Pay";
+"ginivision.digitalinvoice.paybuttontitle" = "Pay %d/%d";
+"ginivision.digitalinvoice.paybuttontitle.accessibilitylabel" = "Pay %d out of %d";
+
+"ginivision.digitalinvoice.items" = "Items %d/%d";
+"ginivision.digitalinvoice.items.accessibilitylabel" = "Items %d out of %d";
+"ginivision.digitalinvoice.whatisthisbutton" = "What is this?";
+
+"ginivision.digitalinvoice.whatisthisactionsheet.title" = "Is it helpful?";
+"ginivision.digitalinvoice.whatisthisactionsheet.message" = "We extracted the detailed items from your invoice to help you calculate the amount to pay in case you want to return some goods or simply not fully-paying it.";
```

Comment:
Sentence is fine, but I had to read it twice... Possibly something a bit more broken up?
"We extracted the detailed items from your invoice. If you would like to return some items or partially pay the invoice, you can deselect them and the amount to pay will be recalculated."

Reply:
Updated.